content/en/docs/ops/ambient/helm-installation/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Installing Ambient Mesh with Helm
+description: How to install Ambient Mesh with Helm.
+weight: 4
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+This guide shows you how to install Ambient Mesh with Helm.
+Besides the demo in [Getting Started with Ambient Mesh](/docs/ops/ambient/getting-started/),
+we **encourage** you to follow this guide to install Ambient Mesh.
+Helm helps you manage components separately, and you can easily upgrade the components to the latest version.
+
+## Prerequisites
+
+1. Perform any necessary [platform-specific setup](/docs/setup/platform-setup/).
+
+1. Check the [Requirements for Pods and Services](/docs/ops/deployment/requirements/).
+
+1. [Install the Helm client](https://helm.sh/docs/intro/install/), version 3.6 or above.
+
+1. Configure the Helm repository:
+
+{{< text bash >}}
+$ helm repo add istio https://istio-release.storage.googleapis.com/charts
+$ helm repo update
+{{< /text >}}
+
+*See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation.*
+
+## Installing the Components
+
+### Installing Base Component
+
+The **Base** chart contains the basic CRDs and cluster roles required to set up Istio.
+This should be installed prior to any other Istio component.
+
+{{< text bash >}}
+$ helm install istio-base istio/base
+{{< /text >}}
+
+### Installing CNI Component
+
+The **CNI** chart installs the Istio CNI Plugin. It is responsible for detecting the pods that belong to the ambient mesh,
```

Comment:
```suggestion
The `cni` chart installs the Istio CNI Plugin. It is responsible for detecting the pods that belong in the ambient mesh,
```

Reply:
I feel like we should also mention its sidecar purpose here... since it isn't only for ambient. This, to me, eludes to the CNI only being used for ambient. Maybe link to the CNI page?

Reply:
yep worth noting this is the same cni from before but with expanded function to configure traffic redirection from app to ztunnel

Reply:
I added the sidecar mode explaination before, however I see the motivation of ambient here https://github.com/istio/istio.io/pull/13871#discussion_r1327736697, which makes sense to me.

Reply:
The suggested change here still is outstanding.